Sources/Core/ModernAVPlayer.swift

```diff
@@ -91,6 +91,7 @@ public final class ModernAVPlayer: NSObject, ModernAVPlayerExposable {
                 config: PlayerConfiguration = ModernAVPlayerConfiguration(),
                 plugins: [PlayerPlugin] = [],
                 loggerDomains: [ModernAVPlayerLoggerDomain] = []) {
+        player.allowsExternalPlayback = false
```

Comment:
`allowsExternalPlayback` value should be fetch from `PlayerConfiguration`

Reply:
Is that really mandatory?
In which case could you possibly have a video output in your lib?

Reply:
no video output for now I'm agree, but it's a player configuration so it should be editable.

Reply:
Its done :)